src/contents/articles/hello-world/index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Hello World
```

Comment:
Yes it's overused but the effect is still there, especially for our target audience.